src/sentry/static/sentry/app/views/onboarding/configure/index.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import Waiting from './waiting';
+import {browserHistory} from 'react-router';
+import ApiMixin from '../../../mixins/apiMixin';
+
+import ProjectContext from '../../projects/projectContext';
+import ProjectDocsContext from '../../projectInstall/docsContext';
+import ProjectInstallPlatform from '../../projectInstall/platform';
+
+const Configure = React.createClass({
+  propTypes: {
+    next: React.PropTypes.func
+  },
+
+  mixins: [ApiMixin],
+
+  componentWillMount() {
+    const {platform} = this.props.params;
+    //redirect if platform is not known.
+    if (!platform || platform === 'other') {
+      this.redirectToNeutralDocs();
+    }
+  },
+
+  componentDidMount() {
+    this.timer = setInterval(() => {
```

Comment:
Should we call`fetchEventData` immediately so that the initial fetch isn't delayed by 2s